### PR TITLE
Probably don't merge; added a few asserts for testing.

### PR DESCRIPTION
**Pull request by chriselrod**

### CODE REVIEW

**Comment by DilumAluthge** [comment]
FWIW I don't think you need this PR any more. CI is all green on `master`. (And code coverage is 100%.)

**Comment by chriselrod** [comment]
I'll rerun this a few times to see if we get another failurel

File: test/runtests.jl [hunk 1]
```diff
@@ -49,28 +42,44 @@ end
...
+        for i = 1:length(ThreadingUtilities.THREADPOOL)
+            ThreadingUtilities.THREADPOOL[i] isa ThreadingUtilities.ThreadTask
+        end
+        if length(ThreadingUtilities.TASKS) > 0
+            @test @inferred(ThreadingUtilities.taskpointer(1)) isa Ptr{UInt}
+        end
+    end
+    
+    @testset "Internals" begin
+        @test ThreadingUtilities._atomic_store!(pointer(UInt64[]), (), 1) == 1
```
Comment (DilumAluthge) [inline test/runtests.jl:66]: @chriselrod You need to change the UInt64 to UInt so that tests will pass with 32-bit Julia binaries.

Comment (chriselrod) [reply test/runtests.jl]: Done

**Pull request closed without merging**